js/main.js

```diff
@@ -0,0 +1,89 @@
+/*jshint esversion: 6 */
+// Book id in localStorage
+if (!localStorage.bookId) {
+  localStorage.bookId = 1;
+}
+// Show Modal
+function toggleBook() {
+  if (document.querySelector('.modal').style.display === 'block')
+    document.querySelector('.modal').style.display = 'none';
+  else {
+    document.querySelector('.modal').style.display = 'block';
+  }
+}
+
+// build library
+let testBook = new Book("Lord Of The Rings", "J.R.Tolkin", 500, false);
+let myLibrary = buildLibrary();
+
+function buildLibrary() {
+  if(localStorage.myLibrary){
+    return JSON.parse(localStorage.getItem("myLibrary")).map( book =>
+      Object.assign(new Book(),book)
+    );
+  }
+  else
```

Comment:
we can avoid this else. Without else simple `return []` will also work 🙂

Reply:
Oh great ! Thanks :)